modules/claims_api/lib/claims_api/evidence_waiver_pdf/pdf.rb

```diff
@@ -52,7 +55,23 @@ def fill_pdf(response)
         page1_options(response),
         flatten: true
       )
+
       @page1_path = temp_path
     end
+
+    def storage_path
+      Rails.root.join('tmp',
+                      "#{SecureRandom.uuid}_final.pdf")
+    end
+
+    def save_pdf(path)
+      output_path = storage_path
```

Comment:
looking at `poa_pdf_constructor/base.rb` and they just use 
```
output_path = "/tmp/#{id}_final.pdf"
```
I know we did this for local, I'm wondering if this is still needed as well? 

I believe we added this when we got the EoF from the certs, thinking it was this file? did we check to make sure this is still required? Assuming it will do the same in production though as from what I remember in local we went to the host machines root without it.